### ISSUE TITLE
Wunsch: Plausi-Prüfung

### ISSUE THREAD

**Issue by KasaFue** [fuerthwiki/wiki#62]
Wäre es möglich, dass wir eine Plausi-Prüfung einrichten könnten. z.B. welche Person lebt schon länger als 100 Jahre, oder ist gestorben bevor sie geboren wurde, oder erhielt einen Preis/Auszeichnung vor der Geburt oder im Babyalter. Habe gerade Volker Heißmann gefunden, der 1969 geboren wurde und 1970 schon die Goldene Bürgermedaille erhielt. Ich denke, wir sollten die Vorteile einer Datenbank auch dahingehend nutzen, dass wir solche Fehler einfach durch eine Plausi-Prüfung der Daten rausfischen.

**Comment by mojoaxel** [comment]
@markmuzenhardt Weis das sicher besser aber ich halte das für absolut möglich. Nur: Denken kann das Ding noch nicht, deshalb wird man jede einzelne Prüfung einzeln basteln müssen...

**Comment by KasaFue** [comment]
FürthWiki als KI    :D .... YES!!!!

**Comment by markmuzenhardt** [comment]
Grundsätzlich ist das möglich. Man müsste dann halt darüber nachdenken, wie man die Anzeige nicht plausibler Daten gestaltet. Ich denke hier an Kategorien, um schon vorhandene aber nicht plausible Artikel zu finden und einer Anzeige in der Faktenbox. Schon im Formular lässt sich sowas noch nicht verhindern, aber eben nach Speicherung.

**Comment by KasaFue** [comment]
Ich dachte, dass man unter den Wartungslisten auf der Spezialseite einfach ein paar Suchen einpflegt, dann kann man hin und wieder einfach mal eine Plausi-Prüfung anschieben.

**Comment by markmuzenhardt** [comment]
Ja, so ungefähr stell ich mir das auch vor. Mit den maschinenlesbaren Datumsangaben ist's ja jetzt auch verhältnismäßig leicht einzubauen. Werde das mal angehen.

**Comment by zonebattler** [comment]
Ich hatte unlängst in die Aktiven-Gruppe geschrieben:

"Der neue Benutzer Karsten Hoffmeyer (kein Fürther, lebt in Berlin, Kamran und ich haben ihn letztes Jahr in Passau kennengelernt) hat gestern Abend (15.09.17) ca. 20 Edits vorgenommen und dabei etliche Fehler beseitigt (z.B. unvollständige Datumsangaben in Formularen). Frage mich, wie er die alle überhaupt so zielstrebig gefunden hat. Gibt's da einen Trick?"

Der Kerl muß doch die unplausiblen Daten irgendwie sehr schnell gefunden haben, und wenn's da einen Befehl bzw. eine Anlistmöglichkeit gibt, wäre das doch schon die halbe Miete, oder?

**Comment by markmuzenhardt** [comment]
Na klar gibts da einen Trick. Semantische abfrage heißt der. Es gibt eine Spezialseite auf der ein Abfragen-Editor liegt. Da kann man sich leicht die plausi Zusammenzimmern.

----- Ursprüngliche Nachricht -----
Von: "zonebattler" <notifications@github.com>
Gesendet: ‎19.‎09.‎2017 09:32
An: "fuerthwiki/wiki" <wiki@noreply.github.com>
Cc: "markmuzenhardt" <mark.muzenhardt@googlemail.com>; "Mention" <mention@noreply.github.com>
Betreff: Re: [fuerthwiki/wiki] Wunsch: Plausi-Prüfung (#62)

Ich hatte unlängst in die Aktiven-Gruppe geschrieben:
"Der neue Benutzer Karsten Hoffmeyer (kein Fürther, lebt in Berlin, Kamran und ich haben ihn letztes Jahr in Passau kennengelernt) hat gestern Abend (15.09.17) ca. 20 Edits vorgenommen und dabei etliche Fehler beseitigt (z.B. unvollständige Datumsangaben in Formularen). Frage mich, wie er die alle überhaupt so zielstrebig gefunden hat. Gibt's da einen Trick?"
Der Kerl muß doch die unplausiblen Daten irgendwie sehr schnell gefunden haben, und wenn's da einen Befehl bzw. eine Anlistmöglichkeit gibt, wäre das doch schon die halbe Miete, oder?
